Workflow/Sources/RenderContext.swift

```diff
@@ -180,12 +180,17 @@ protocol RenderContextType: AnyObject {
 extension RenderContext {
     public func makeSink<Event>(
         of eventType: Event.Type,
-        onEvent: @escaping (Event, inout WorkflowType.State) -> WorkflowType.Output?
+        onEvent: @escaping (
```

Comment:
- [ ] TODO: resolve this

Reply:
just reverted it for now. we can make an additive change if we need to in the future